packages/vue/src/composables/useClerkContext.ts

```diff
@@ -0,0 +1,15 @@
+import { inject } from 'vue';
+
+import { ClerkInjectionKey } from '../keys';
+
+export function useClerkContext() {
+  const ctx = inject(ClerkInjectionKey);
+
+  if (!ctx) {
+    throw new Error(
+      'This component/composable can only be used when the Vue plugin is installed. Learn more: https://clerk.com/docs/components/clerk-provider',
```

Comment:
Shouldn't this link to the installation docs of the Clerk Vue SDK instead?

Reply:
Updated to add a quickstart link to Vue SDK (docs to follow)